src/ranged.cpp

```diff
@@ -71,15 +71,98 @@ size_t blood_trail_len( int damage )
+// projectile_attack represents the possible error
+// due to imperfections in the weapon and the
+// steadiness of the shooter's aim; for the same
+// weapon / shooter / recoil you'll have the same
+// dispersion.
+//
+// The path each shot takes has some random error.
+// This is modelled as a normal distribution
+// centered around the aiming point, or - equivalently -
+// as an error angle (shot_dispersion) that is centered
+// around zero. A shot_dispersion of zero is a
+// perfectly-on-target shot. shot_dispersion, unlike
+// dispersion, is a signed value, the error might be
+// to the left (negative) or right (positive).
+//
+// The standard deviation of the normal distribution
+// is (dispersion / dispersion_sigmas). With the current
+// value of dispersion_sigmas (2.4) this means that
+// the dispersion value is 2.4 standard deviations
+// and 98% of all shots will lie within the
+// dispersion angle.
+//
+// Both dispersion and shot_dispersion are measured in
+// arcminutes (1/60th of a degree).
+//
+// Increasing dispersion_sigmas will decrease the standard
+// deviation, shots will be "tighter" around the actual
+// target, the overall chance to hit increases.
+//
+// Allowing a hit on the target just because you
+// hit the right tile doesn't work balance-wise as
+// it's too big of a target if you also want to keep
+// miss angles within a reasonable range. So there is
+// a second parameter, occupied_tile_fraction, which says
+// "how wide" the target is. You have to get at least this
+// close with your shot, in tiles (missed_by_tiles)
+// to have a chance to hit. Increasing
+// occupied_tile_fraction increases the chance to hit.
+//
+// You should not set occupied_tile_fraction > 1.0 as
+// that would mean that hitting an adjacent tile is enough
+// to hit the critter, and the code that follows the shot
+// trajectory looking for hits is not capable of handling
+// that.
+//
+// Once you are close enough with the shot for a hit,
+// exactly how close influences the quality of the hit.
+// missed_by represents how good the hit is on a scale
+// of 0 to 1; 0.0 is a perfect hit; 0.99999.. is the
+// worst possible shot that still connects (where you
+// got just barely within occupied_tile_fraction tiles
+// of where you aimed and just clipped the edge of your
+// target)
+//
+// dispersion_sigmas and occupied_tile_fraction can be
+// tuned together to affect the spread of misses without
+// affecting hit or hit quality chances, e.g. if you
+// increase dispersion_sigmas by 10% and also decrease
+// occupied_tile_fraction by 10%, then the cone of misses
+// will get 10% tighter but the overall hit and hit
+// quality chances are unchanged.
+
+// these parameters balance the hit/miss chance:
+const double dispersion_sigmas = 2.4;      // how many standard deviations does "dispersion" represent?
+const double occupied_tile_fraction = 0.5; // how much of the target tile is occupied by the target?
```

Comment:
Can some of them be moved inside functions? If they can't my gut feeling is that dispersion_sigma belongs in `game_constants.h` as it is after all a game constant.